includes/defaults.inc.php

```diff
@@ -631,6 +631,8 @@
 $config['auth_ldap_emailattr']                  = 'mail';
 $config['auth_ldap_cache_ttl'] = 300;
 // How long in seconds should ldap* module cache user information in $_SESSION
+$config['auth_ldap_userdn']                     = true;
```

Comment:
This changes the default for all installs.

Should be set to `false` to maintain backwards compatibility.

Reply:
Done!